common/src/main/java/org/broadleafcommerce/common/persistence/EntityDuplicatorImpl.java

```diff
@@ -136,7 +138,7 @@ public boolean validate(final Object entity) {
 
     @Transactional(TransactionUtils.DEFAULT_TRANSACTION_MANAGER)
     @Override
-    public <T> T copy(final Class<T> entityClass, final Long id) {
+    public <T> T copy(final Class<T> entityClass, final Long id, final String catalogIdentifier) {
```

Comment:
think we can leave as it is and do  BroadleafRequestContext.getBroadleafRequestContext().getAdditionalProperties().put("duplicateToCatalogId", catalogIdentifier); in MultiTenantAdminBasicEntityController for example.
Logic is that some clients don't use MT module but will have logic belonging to MT and this will not work I think